src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -2564,11 +2602,11 @@ TEST_P(conv_fp32_reorder_bfyx_to_fsv32_conv_basic, basic) {
     execute(p);
 }
 INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_fp32_reorder_bfyx_to_fsv32_conv_basic, ::testing::ValuesIn(std::vector<convolution_test_params>{
-    convolution_test_params{ FSV32_CASE_CONV_FP32_1,  3, 3 }
+    convolution_test_params{ FSV32_CASE_CONV_FP32_1,  3, 4 }
```

Comment:
Is reorder_out being fused to convolution in the above TC? if so.. the # of fused network should be 2?

Reply:
reorder_out is not fused. only reorder_fsv32 is fused.

Reply:
If so, this test is not for "fused reorder which is converting fsv conv output layout to bfyx layout" ?

Reply:
This test is for "fused reorder which is converting bfyx to fsv32, in front of convolution"